bigbluebutton-html5/imports/ui/components/user-list/chat-list-item/component.jsx

```diff
@@ -1,22 +1,67 @@
 import React, { Component } from 'react';
+import UserAvatar from '/imports/ui/components/user-avatar/component';
+import Icon from '/imports/ui/components/icon/component';
 import styles from './styles.scss';
 import { Link } from 'react-router';
-import classNames from 'classnames';
 
-export default class ChatListItem extends Component {
+const PRIVATE_CHAT_PATH = 'users/chat/';
+
+const propTypes = {
+  chat: React.PropTypes.shape({
+    id: React.PropTypes.string.isRequired,
+    name: React.PropTypes.string.isRequired,
+  }).isRequired,
+};
+
+const defaultProps = {
+};
+
+class ChatListItem extends Component {
   render() {
+    const {
+      chat,
+    } = this.props;
+
+    const linkPath = [PRIVATE_CHAT_PATH, chat.id].join('');
+    let fakeUnreadCount = Math.round(Math.random() * 33);
+
     return (
-      <li tabIndex='0' className={styles.chatListItem} {...this.props}>
-        <div className={styles.chatThumbnail}>
-          <i className="icon-bbb-group-chat"></i>
-        </div>
-        <div className={styles.chatName}>
-          <h3 className={styles.chatNameMain}>Public Chat</h3>
-        </div>
-        <div className={styles.unreadMessages}>
-          <p className={styles.unreadMessagesText}>{Math.round(Math.random() * 33)}</p>
-        </div>
+      <li {...this.props}>
+        <Link to={linkPath} className={styles.chatListItem}>
+          {chat.icon ? this.renderChatIcon() : this.renderChatAvatar()}
+          <div className={styles.chatName}>
+            <h3 className={styles.chatNameMain}>{chat.name}</h3>
+          </div>
+          <div className={styles.unreadMessages}>
+            <p className={styles.unreadMessagesText}>{fakeUnreadCount}</p>
+          </div>
+        </Link>
       </li>
     );
   }
+
+  renderChatAvatar() {
+    const {
+      chat,
+    } = this.props;
+
+    return <UserAvatar user={chat}/>;
+  }
+
+  renderChatIcon() {
+    const {
+      chat,
+    } = this.props;
```

Comment:
What's the benefit of duplicating these 3 lines 3 times in the component instead of just accessing it as `this.props.chat` ? You don't access this variable more than once anywhere.